paywall/paywall-builder/constants.js

```diff
@@ -0,0 +1,4 @@
+export const LOCKED = 'locked'
```

Comment:
Hum, I am sorry to ask you this but could we use more descriptive constants? Like `POST_MESSAGE_LOCKED` for example?

Reply:
done, PTAL